presentation/components/src/test/instance-filter-builder/PresentationInstanceFilterBuilder.test.tsx

```diff
@@ -63,9 +95,24 @@ describe("PresentationInstanceFilter", () => {
       return;
     }
     imodelMock.setup((x) => x.query(moq.It.isAnyString(), moq.It.isAny(), moq.It.isAny())).returns(() => generator());
+    imodelMock.setup((x) => x.key).returns(() => "test_imodel");
+    imodelMock.setup((x) => x.onClose).returns(() => onCloseEvent);
+    const metadataProvider = getIModelMetadataProvider(imodelMock.object);
+    sinon.stub(metadataProvider, "getECClassInfo").callsFake(async (id) => {
+      switch (id) {
+        case classInfo.id:
+          return new ECClassInfo(classInfo.id, classInfo.name, classInfo.label, new Set(), new Set());
+      }
+      return undefined;
+    });
   });
 
-  it("invokes 'onInstanceFilterChanged' with filter", async () => {
+  afterEach(() => {
+    onCloseEvent.raiseEvent();
+    imodelMock.reset();
+  });
+
+  it("invokes 'onInstanceFilterChanged' without filter", async () => {
```

Comment:
Looks like name of the test does not match test body. I suggest leaving old name.